src/NServiceBus.Core/ObjectBuilder/ServiceCollection.cs

```diff
@@ -0,0 +1,95 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
```

Comment:
we need to document modifications made to this file (made all public types internal and changed namespace)

Reply:
It's on the to-do list